pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -1536,6 +1536,8 @@ func SortVHostRoutes(routes []*route.Route) []*route.Route {
 // isCatchAllRoute returns true if an Envoy route is a catchall route otherwise false.
 func isCatchAllRoute(r *route.Route) bool {
 	catchall := false
+	// A Match is catch all if and only if it has no header/query param match
+	// and URI has a prefix / or regex .*.
```

Comment:
I think it would be better to omit the last `.`, or add a backtick.

Reply:
Although I think it looks ok this way.

Reply:
HHA